### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by mahfoudfx** [laravel/nova-issues#4262] [General]
Hello,
Please find below my wishlist for some UI and fields features that I estimate to be "Core Features":

1. Autofocus/Default focus on a specified field on opening (Unless I'm missed it in documentation).
2. Tabs (https://novapackages.com/packages/eminiarts/nova-tabs).
3. Wizards (https://novapackages.com/packages/zareismail/nova-wizard).
4. Country/Dial phone code/Currency field with flags (https://symfony.com/bundles/EasyAdminBundle/current/fields/CountryField.html).
5. Menu Badges/Notifications (https://easycorp.github.io/blog/posts/new-in-easyadmin-4-menu-badges).
6. RTL support (RightToLeft).
7. As Nova became responsive, PWA support will be a serious plus.

Good Luck for your great work.

**Comment by mahfoudfx** [comment]
Hello again,
I forgot to mention the Breadcrumbs feature (https://novapackages.com/packages/chris-ware/nova-breadcrumbs).

**Comment by piljac1** [comment]
For the autofocus functionality, you can use:
```php
->withMeta([
    'extraAttributes' => [
        'autofocus' => true
    ]
])
```
However, it doesn't work on all fields, but works well on text based fields (Text, Email, etc.).
